protocol-units/execution/opt-executor/src/executor/transaction_pipe.rs

```diff
@@ -158,6 +176,56 @@ mod tests {
 		Ok(())
 	}
 
+	#[tokio::test]
+	async fn test_pipe_mempool_with_malformed_transaction() -> Result<(), anyhow::Error> {
```

Comment:
@mzabaluev This should error. I don't know why it doesn't. It's two of the same transaction being submitted to the mempool.

Reply:
Use `#[ignore]` as a maintenance-friendly way to mark a test that does not do what's expected, so it sticks out to anyone who would look at the test summaries later.